docs/source/03_HED_formats.md

```diff
@@ -129,7 +129,7 @@ The location of the node element within the section specifies its relationship t
 A node element specifies a name,
 node attributes, and an informative description of the tag term's meaning.
 A node name may only contain valid `name` characters 
-(alphanumeric, blank, hyphen, underscore, period, and non-ascii).
+(alphanumeric, hyphen, underscore, period, and nonascii).

```

Comment:
"nonascii" is not a word. Arguably, all of these character classes should be in code syntax as a result.

Reply:
Fixed PR#573